<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="browse1">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browsepath1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select template:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select authentication links:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browsepath2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkboxall">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>64</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkboxcustom">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>64</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="runbutton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount of links:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="customline">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="loadingline">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output folder:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="browsepath3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
